### PR TITLE
Replacing the illustrations

### PR DESCRIPTION
**Pull request by chloe0592**
Connected to this PR - [here](https://github.com/commercetools/merchant-center-frontend/pull/14234).

### CODE REVIEW

File: packages/assets/images/icecream.svg
Comment (emmenko) [inline packages/assets/images/icecream.svg:1]: Let's keep the old illustrations, as otherwise it's a breaking change

**Review by emmenko — COMMENTED** [review]
🤗

File: packages/assets/images/icecream.svg
Comment (chloe0592) [reply packages/assets/images/icecream.svg:1]: Good catch then! Thank you 🙇‍♀️  - [59b1c01](https://github.com/commercetools/merchant-center-application-kit/pull/3017/commits/59b1c0193af34ffe32784232edd285b699fecc77)

File: .changeset/smart-avocados-smile.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+"@commercetools-frontend/application-components": patch
+"@commercetools-frontend/application-shell": patch
+"@commercetools-frontend/assets": patch
+"@commercetools-local/visual-testing-app": patch
+---
+
+Update illustrations.
```
Comment (emmenko) [inline .changeset/smart-avocados-smile.md:8]: We can also mention maybe which illustrations (by file name) are supposed to be deprecated.

**Review by emmenko — APPROVED** [review]

**Comment by emmenko** [comment]
I think @culotta should also approve these.

File: .changeset/smart-avocados-smile.md (hunk 1 above)
Comment (chloe0592) [reply .changeset/smart-avocados-smile.md]: Done here - https://github.com/commercetools/merchant-center-application-kit/pull/3017/commits/d9aa1320f66d0b4c590d382ab169a13bcd6bdff3 ✅

**Review by culotta — APPROVED** [review]

**Pull request merged**